utils/wasm-proc/metadata-js/test.js

```diff
@@ -25,3 +25,25 @@ wasmMetadata.getWasmMetadata(wasmBytes).then(metadata => {
         }
     )
 });
+
+// async
+wasmBytes = fs.readFileSync(
+    path.join(__dirname, "../../../examples/target/wasm32-unknown-unknown/release/demo_async.meta.wasm")
```

Comment:
You also need add `wasm-proc` execution for `demo_async` program in `./scripts/meta-test.sh` and CI